CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 15.0.0
+
+### Features
+No 15 packages were published. This update is a stepping stone to a 16.0.0 version.
```

Comment:
Update README. Follow what we put for 14.0.0 regarding upgrading Angular version

Reply:
Edited with same format.